packages/babel-core/test/helpers/esm.js

```diff
@@ -55,7 +55,7 @@ async function spawn(runner, filename, cwd = process.cwd()) {
   );
 
   const EXPERIMENTAL_WARNING =
-    /\(node:\d+\) ExperimentalWarning: The ESM module loader is experimental\./;
+    /\(node:\d+\) ExperimentalWarning: (The ESM module loader is experimental\.|Support for loading ES Module in require\(\) is an experimental feature and might change at any time\n\(Use `node --trace-warnings ...` to show where the warning was created\))/;
```

Comment:
Note that our users will, as far as I know, not see this warning because it's automatically suppressed when inside `node_modules`.